src/test/java/com/oltpbenchmark/catalog/TestCatalog.java

```diff
@@ -60,8 +60,7 @@
 //        for (Entry<String, String> e : origTableNames.entrySet()) {
 //            assertFalse(e.toString(), e.getKey().equals(e.getValue()));
 //            assertTrue(e.toString(), e.getKey().equalsIgnoreCase(e.getValue()));
-//        } // FOR
-//    }
+//        }//    }
```

Comment:
I guess we used a find and replace regex and it probably should have preserved the line break during the replace; this (and another test file below) pulled the comment upwards, which can potentially break if block-uncommented like in this scenario. But it probably doesn't really matter.

Reply:
hehe.  i'm desperately want to delete all this commented out code.  thats what version control is for :)

Reply:
I personally have no objections to deleting dead code, if @apavlo doesn't mind.